client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -315,6 +374,19 @@ class CampaignAdminList extends SilverStripeComponent {
     return groups;
   }
 
+  getPlaceholderGroups() {
```

Comment:
I'm not sure why we need this api; doesn't groupItemsForSet() build the groups automatically?